.eslintrc

```diff
@@ -1,5 +1,10 @@
 {
 	"extends": [
 		"@quartz/eslint-config-base"
+	],
+	"overrides": [
+		{
+			"files": [ "*.js", "*.ts" ],
+		}
```

Comment:
It doesn't look like this overrides is actually overriding anything

Reply:
Oh weird, I thought I had to include this in order for it to run on .ts files, but it seems to work without it